src/app/public/modules/phone-field/types/country-data.ts

```diff
@@ -0,0 +1,6 @@
+export interface SkyCountryData {
```

Comment:
Let's name this `SkyPhoneFieldCountry` so that there's no confusion as to which module it belongs to. (Also, think we could remove the suffix `Data`? It feels extraneous.)

Reply:
Done